doc/source/how-to/documenting.rst

```diff
@@ -469,349 +470,210 @@ branch within this repository. This is done by uploading the generated
 
-The following job step shows the logic for deploying. If you want to deploy to
-another repository, make sure to uncomment the ``repository-name`` line and
-declare the name of your documentation repository:
 
-.. tab-set::
-
-    .. tab-item:: Using ``tox``
-
-        .. code-block:: yaml
-
-            - name: Deploy
-              if: github.event_name == 'push' && contains(github.ref, 'refs/tags')
-              uses: JamesIves/github-pages-deploy-action@v4.3.0
-              with:
-                token: ${{ secrets.GITHUB_TOKEN }}
-                # repository-name: pyansys/repository-name
-                branch: gh-pages
-                folder: .tox/doc_out
-                clean: true
-
-    .. tab-item:: Without using ``tox``
-
-        .. code-block:: yaml
-
-            - name: Deploy
-              if: github.event_name == 'push' && contains(github.ref, 'refs/tags')
-              uses: JamesIves/github-pages-deploy-action@v4.3.0
-              with:
-                token: ${{ secrets.GITHUB_TOKEN }}
-                # repository-name: pyansys/repository-name
-                branch: gh-pages
-                folder: doc/_build/html
-                clean: true
-
-
-Notice that for previous job steps, a ``GITHUB_TOKEN`` is required. GitHub
-automatically generates the token ``GITHUB_TOKEN`` which you can use as
-``token`` for deploying documentation to the same repository. However, if you are
-planning to deploy to another repository, this token does not have the necessary permissions.
+Deploying to ``gh-pages`` of the repository
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+While you could manually upload your auto-generated documentation for each
+release using your own local GitHub credentials, the best practice is to have
+your documentation build on GitHub and deployed either on release or push to
+main. You can do this via `GitHub Actions`_ by taking advantage of
+``pyansys/actions`` reusable workflows.
 
-In this case, there are two options for documentation deployment, using a bot
-and using a personal access token (PAT). Depending on your profile (such as how many
-organizations you work in and your permissions for different repositories), using
-a PAT can be potentially dangerous because a PAT is not restricted to defined
-repositories but rather has general permissions. This means that a PAT with
-`repository-write` permission can write in any repository in any organization that the
-PAT creator can access.
+Generated documentation gets published in the ``gh-pages`` branch of the
+repository of the project. On top of that, two artifacts named
+``documentation-html`` and ``documentation-pdf`` get generated at the end of the
+workflow.
 
-Therefore, the recommended approach is to **use a bot**. However, you can use a
-PAT if you feel it fits your needs better.
+Add the following workflow job to build and deploy development and stable
+documentation in an automated way.
 
+.. code-block:: yaml
```

Comment:
Of course, if you make my suggested name changes from earlier, they will also need to be made in the subsequent lines.